lib/settings/pages/video_player_settings.dart

```diff
@@ -0,0 +1,188 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:thunder/core/enums/local_settings.dart';
+import 'package:thunder/core/enums/video_auto_play.dart';
+import 'package:thunder/core/enums/video_playback_speed.dart';
+import 'package:thunder/core/singletons/preferences.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:thunder/settings/widgets/list_option.dart';
+import 'package:thunder/settings/widgets/toggle_option.dart';
+import 'package:thunder/utils/bottom_sheet_list_picker.dart';
+
+class VideoPlayerSettingsPage extends StatefulWidget {
+  const VideoPlayerSettingsPage({super.key, this.settingToHighlight});
+
+  final LocalSettings? settingToHighlight;
+
+  @override
+  State<VideoPlayerSettingsPage> createState() => _VideoPlayerSettingsPageState();
+}
+
+class _VideoPlayerSettingsPageState extends State<VideoPlayerSettingsPage> {
+  /// Loading
+  bool isLoading = true;
+
+  LocalSettings? settingToHighlight;
+  GlobalKey settingToHighlightKey = GlobalKey();
+
+  /// Toggle to always start video in fullscreen landscape when enabled
+  bool videoAutoFullscreen = true;
```

Comment:
Some of these default don't seem the match the way you've set them elsewhere (i.e., ThunderBloc).